### ISSUE TITLE
[Menu] Tooltip doesn't work properly when set on the `sl-menu-button`

### ISSUE THREAD

**Issue by anna-lach** [sl-design-system/components#2971]
### Provide a general summary of the issue here

Tooltip doesn't work properly when set on the `sl-menu-button`.

The focusable element in the `sl-menu-button` is a `sl-button` so that button should be connected with the tooltip via `aria-describedby` or `aria-labelledby`.

Please check: https://github.com/sl-design-system/components/issues/2748#issuecomment-3811429972
<img width="833" height="379" alt="Image" src="https://github.com/user-attachments/assets/816f21ef-e6a1-446e-a262-5f02affb20ea" />

### 👤 Your name

Anna Sobczak

### 🧢 Your Product/Team

SLDS
